configure.ac

```diff
@@ -69,20 +69,20 @@ AS_CASE([$PYTHON_SHEBANG],
 AC_SUBST([PYTHON_SHEBANG])
 
 AC_ARG_ENABLE([html],
-  AS_HELP_STRING([--disable-html], [don't build and install HTML docs]),
+  AS_HELP_STRING([--disable-html], [HTML documentation]),
```

Comment:
Not a big deal, but is this relevant to the overall PR?

Reply:
Yes, kind of in a roundabout way though. Help for one of the other `--disable` options got too long, so I removed "don't build and install" to make it fit, and changed the others for consistency.

Reply:
Fair enough